src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +47,30 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
move formatter to class: [https://mate-academy.github.io/style-guides/java/java.html](url)

Reply:
sorry, page not found. Link don't work.